src/Evolution/Ringdown/StrahlkorperCoefsInRingdownDistortedFrame.cpp

```diff
@@ -91,17 +128,16 @@ std::vector<DataVector> strahlkorper_coefs_in_ringdown_distorted_frame(
       static_cast<size_t>(5),
       false,
       std::nullopt,
-      {100.0},
+      {50.0},
```

Comment:
Add comments as to why 50 and not 100 (or some other number).  Same for other magic numbers here.